lib/task.js

```diff
@@ -967,4 +972,29 @@ export class Task {
     }
     return success;
   }
+
+  /**
+  Create a new queue using temp credentials.
+
+  @param {credentials} Temporary credentials.
+  @param {runtime} Runtime object.
+
+  @return New queue.
+  */
+  createQueue(credentials, runtime) {
+    return new taskcluster.Queue({
+      credentials: credentials,
+      stats: base.stats.createAPIClientStatsHandler({
```

Comment:
Because we need a queue client per claim we're doing (coalesced tasks), I'm not sure if we really need stats from the secondary claims it's making, just for the primary one should be good.